.github/workflows/update-connectors-list.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Update connectors' list"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 8 * * 1'
+
+jobs:
+  update-connectors-list:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Update connectors' list
+        working-directory: src/connectorgen
+        run: |
+          go run main.go ${GITHUB_WORKSPACE}/static/connectors.json
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6.0.2
```

Comment:
Can we also auto-merge the PR? No need to have the safety check (that's future work), but auto merging would be nice to keep the manual toil to a minimum, otherwise we'll soon be annoyed by this.

I didn't look too far into this, but I see a similar action that should allow us to do that [peter-evans/enable-pull-request-automerge](https://github.com/peter-evans/enable-pull-request-automerge).

Reply:
Hmm, I guess that's fine... After all, we can easily rebuild the list or revert the change (if needed).

Reply:
Done!